### DISCUSSION TITLE
1.2.2

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#27] [Announcements]
1.  更新文件管理器实现
2. 判断用户是否安装了 quickstart , 避免非 istoreos 用户点击高级设置跳转到错误页面
3. 优化UI

**Comment by Fengerxuanxiao** [comment]
👍，这个版本的文件管理好用多了，
已知问题：好像在安装路由狗之前所安装的插件无法打开管理页面，在安装路由狗之后的插件就没有问题。
建议：就是自定义图标如果能够添加一个自定义图标样式或者自动搜刮网页图标就好了。这样的话，我觉得做浏览器标签页都完全没问题了
大佬辛苦了🫡

**Reply by Carseason** [reply]
https://github.com/Carseason/routerdog/discussions/30

**Comment by tizips** [comment]
大佬，目前的文件上传，不支持多文件上传，还需要切换到 iStoreOS 那边操作。后续有实现的想法吗

**Reply by Carseason** [reply]
https://github.com/Carseason/routerdog/discussions/30
